web/router.ex

```diff
@@ -11,9 +11,7 @@ defmodule RemoteRetro.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    if Application.get_env(:remote_retro, :add_mock_user_to_session) do
-      plug :add_mock_user_to_session
-    end
+  
```

Comment:
@pulsedemon @Zanadar awesome change (I think?), but not quite sure how it works. don't you get redirected to authenticate with google if the user isn't on session? Or am I missing something?